pkg/asset/installconfig/openstack/realvalidvaluesfetcher.go

```diff
@@ -24,25 +24,23 @@ func NewValidValuesFetcher() validation.ValidValuesFetcher {
 
 // GetCloudNames gets the valid cloud names. These are read from clouds.yaml.
 func (f realValidValuesFetcher) GetCloudNames() ([]string, error) {
-	clouds, err := new(yamlLoadOpts).LoadCloudsYAML()
+	clouds, err := clientconfig.LoadCloudsYAML()
 	if err != nil {
 		return nil, err
 	}
 
-	i := 0
 	cloudNames := make([]string, len(clouds))
 	for k := range clouds {
-		cloudNames[i] = k
-		i++
+		cloudNames = append(cloudNames, k)
 	}
 	return cloudNames, nil
 }
 
 // GetNetworkNames gets the valid network names.
 func (f realValidValuesFetcher) GetNetworkNames(cloud string) ([]string, error) {
-	opts := defaultClientOpts(cloud)
```

Comment:
defaultClientOpts adds modified cloud loaders to the default opts: https://github.com/openshift/installer/blob/1cc6935753842458bb84504f8e247ddc86cb93c7/pkg/asset/installconfig/openstack/session.go#L36